### PR TITLE
PHP sdk extension

### PR DESCRIPTION
**Pull request by stobbsm**
I've been needing to write a lot of php lately, and noticed this didn't exist yet.

I kept it as php 7.3, as others may require different versions later.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build org.freedesktop.Sdk.Extension.php73

**Comment by bilelmoussaoui** [comment]
Why someone would need PHP as a flatpak anyway?

**Comment by ZanderBrown** [comment]
Some people write desktop apps in PHP see [phpdesktop](https://github.com/cztomczak/phpdesktop/) or [php-gtk](http://gtk.php.net/)

File: org.freedesktop.Sdk.Extension.php73.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+					"type": "file",
+					"path": "php.ini-development",
+					"dest-filename": "php.ini"
+				}
+			]
+		},
+		{
+			"name": "composer",
+			"build-options": {
+				"build-args": ["--share=network"]
```
Comment (bilelmoussaoui) [inline org.freedesktop.Sdk.Extension.php73.json]: Builder don't have access to network after downloading the sources. This won't work here on Flathub.

**Comment by bilelmoussaoui** [comment]
The PR to should be made from your branch to new-pr branch.

**Comment by stobbsm** [comment]
> Why someone would need PHP as a flatpak anyway?

To use it in things like atom and vscode when writing php.

File: org.freedesktop.Sdk.Extension.php73.json (hunk 1 above)
Comment (stobbsm) [reply org.freedesktop.Sdk.Extension.php73.json]: I'll try to work it out without that then.

**Comment by stobbsm** [comment]
> The PR to should be made from your branch to new-pr branch.

Whoops, completely forgot that. Will close this an resubmit when I've solved the composer issue.

**Pull request closed without merging**